src/js/components/__tests__/ServiceInfo-test.js

```diff
@@ -52,13 +52,13 @@ describe('ServiceInfo', function () {
 
     it('renders health state', function () {
       expect(
-        this.node.querySelectorAll('.page-header-text')[0].children[1].children[0].textContent
+        this.node.querySelectorAll('.page-header-sub-heading')[0].children[0].children[0].textContent
```

Comment:
#nit: If you are just selecting the first anyway, you can just use `querySelector` (i.e. no `All`)